File: 7-kyu/make-a-function-that-does-arithmetic!/python/solution.py
def add(a, b):
    return a + b

def subtract(a, b):
    return a - b

def multiply(a, b):
    return a * b

def divide(a, b):
    return a / b

def arithmetic(a, b, operator):
    lookup_table = { "add": add(a, b), "subtract": subtract(a, b), "multiply": multiply(a, b), "divide": divide(a, b) }
    return lookup_table[operator]

File: 7-kyu/make-a-function-that-does-arithmetic!/javascript/solution.js
const arithmetic = (a, b, operator) => {
  const lookup = {add: (x, y) => x + y, subtract: (x, y) => x - y,
                  multiply: (x, y) => x * y, divide: (x, y) => x / y};
  return lookup[operator](a, b);
};
